flow/core/kernel/vehicle/traci.py

```diff
@@ -747,7 +750,7 @@ def _multi_lane_headways(self):
                 for lane in range(max_lanes):
                     edge_dict[edge][lane].sort(key=lambda x: x[1])
 
-        for veh_id in self.get_rl_ids():
+        for veh_id in self.get_ids():
```

Comment:
this could seriously slow down runtime. can we make sure that is not happening before we change it?